### ISSUE TITLE
[Feature Request] Adding thumbnail preview to lora list (not sure if possible)

### ISSUE THREAD

**Issue by aphaits** [pythongosssss/ComfyUI-Custom-Scripts#13]
The lora folder listing is really helpful but I thought it would be extra helpful if you can show a thumbnail when you hover next to the lora you are trying to pick from the list. I have no idea if this is possible or not but its really going to help a lot if you have tons of loras in multiple folders.

This is how I usually save thumbnail with my loras or other checkpoint models, same filename but jpg.
![image](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/assets/132622276/86f3eed0-87a5-43a5-941e-37af16ab8230)

The folder list is already so much helpful in selecting loras
![image](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/assets/132622276/5b22b098-8a1b-4843-b289-0f9a7449ae24)

I'm thinking of something like a small thumbnail of the jpg can show up when you mousehover the lora list
![image](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/assets/132622276/ac361c32-2dc4-4f43-bdf7-d2077f4e2c3f)

**Comment by pythongosssss** [comment]
I've added this, the way it is implemented isnt compatible with the standard Lora node, so there is a new Lora Loader that functions exactly the same as the standard one, just with the addition of images (a * at the end indicates there is an image)

![image](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/assets/125205205/4a0a5244-2201-4c1f-b5f0-7ac18eacb19a)    

Images must be named the same as the lora and .png or .jpg  
Currently subfolders are incompatible with this, i'll need to think of a way to get them to work.

If you have any issues let me know

**Comment by abdullahcfix** [comment]
>(a * at the end indicates there is an image)

Hi, I have a couple Loras that do not have the * after their name and a preview image does not appear when hovering over them, though the rest of them do. Those Loras are still on Civitai, so not deleted at all.

I did notice that they are ones I downloaded recently, like in past 2-3 days while the rest that do have previews are from a month ago. Is there like a grace period or something that has to occur before previews become available? I restarted ComfyUI and it didn't solve the problem.

Also, when these previews are retrieved, are they being read directly from Civitai's servers each time you hover on a Lora, or are they being cached somewhere locally? If they're being cached, where and for how long? Back when I used to use SD.Next for SD1.5 and SDXL, I had the Civitai Helper extension that would download previews and save them locally, so in the even that someone deletes their model/Lora from Civitai, I'd still have the preview from before. Just wondering if it works the same for this or not.

Sorry for all the questions or if there's an easy solution for it, I'm still a little new to ComfyUI and nodes in general.

Thanks.
